networking_ccloud/tests/unit/db/test_db_plugin.py

```diff
@@ -34,6 +35,17 @@ class TestDBPluginNetworkSyncData(test_segment.SegmentTestCase, base.PortBinding
     def setUp(self):
         super().setUp()
 
+        cfg.CONF.set_override('global_physnet_mtu', 9000)
+        cfg.CONF.set_override('path_mtu', 9000, group='ml2')
+        cfg.CONF.set_override('network_vlan_ranges', ['foo:100:410', 'bar:200:210', 'baz:300:310',
+                                                      'spam:500:510', 'ham:600:610', 'mew:700:710',
+                                                      'caw:800:810'],
+                              group='ml2_type_vlan')
```

Comment:
If you introduce a config override, I think you should also have a cleanup for that.

Reply:
You mean for each time I have a config override somewhere? I think in general each module should override the things it needs to be a certain way and with that not much cleanup should be needed.

On the other hand I might still be a bit emotional about this, as I really expected config overrides to not be shared between tests.